content/applications/sales/sales/send_quotations/create_quotations.rst

```diff
@@ -0,0 +1,337 @@
+=================
+Create quotations
+=================
+
+In Odoo *Sales*, quotations can be quickly created and sent to customers. Once a quotation has been
+confirmed, it officially turns into a *sales order*, which can then be invoiced and paid for.
+
+.. _sales/quotation-settings:
+
+Quotation settings
+==================
+
+While the ability to create and send quotations in the Odoo *Sales* application is avaialble by
+default, there are a collection of quotation and order settings that can be customized and
+configured, if needed.
+
+To access these setting options, navigate to :menuselection:`Sales app --> Configuration -->
+Settings`, and scroll to the :guilabel:`Quotations & Orders` section.
+
+.. image:: create_quotations/quotations-orders-section.png
+   :align: center
+   :alt: The Quotations and Orders section on the Odoo Sales app Settings page.
+
+- :guilabel:`Quotation Templates`: create unique quotation templates featuring standardized offers
```

Comment:
All of the "to learn more" items in this list can just be accumulated in a **seealso** admonition at the end, or the links can be more integrated with the list item descriptions as inline links or individual **seealso**s; we don't need to say over and over "to learn more go to the x documentation"

Reply:
Understood. Moved them to the seealso at the end of the doc. Thanks for the super helpful suggestion. I really appreciate the attention to detail. 🙏